.github/workflows/tests.yml

```diff
@@ -0,0 +1,52 @@
+name: Tests
+
+on:
+    push:
+    pull_request:
+
+jobs:
+
+    redis-tests:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                php: [7.2, 7.3, 7.4]
+        services:
+            redis:
+                image: redis
+                ports:
+                    - 6379:6379
+                #options: --entrypoint redis-server
+            redis-cluster:
+                image: grokzen/redis-cluster:4.0.8
+                ports:
+                    - 7000:7000
+                    - 7001:7001
+                    - 7002:7002
+                    - 7003:7003
+                    - 7004:7004
+                    - 7005:7005
+        container:
+            image: jakzal/phpqa:php${{ matrix.php }}
```

Comment:
I only used this image as a proof of concept. We need to agree which image to use (or create our own one?).

Reply:
There's also an option to run directly on ubuntu-latest from github, which already has php 7.1-7.4 baked in (some extensions might be missing).

Reply:
hello @jakzal , did you had a look at https://github.com/shivammathur/setup-php?

Reply:
I came across it but didn’t get a chance to use it yet. Indeed, it looks like a good option.